tests/test_endpoints/test_event.py

```diff
@@ -219,6 +247,72 @@ def test_get_event_participants(client):
     assert 'food' in res_json[0]
 
 
+@authentication_required("/api/event/{uuid}/options", "get")
+def test_get_event_options(client):
+    # Login
+    access_token = client_login(
```

Comment:
The login doesn't return an access token anymore, so this and the headers should just be removed like you did on `test_get_joined_events`